### DISCUSSION TITLE
Sample Usage / Documentation in the github repo

### DISCUSSION THREAD

**Discussion by antoponselvan** [maniac-tech/react-native-expo-read-sms#33] [General]
Great work! Thanks for solving this pain point.
Just a minor suggestion. Please consider linking the Medium article you have written explaining how to use this library. It was not very obvious to me on what are the arguments that the callback functions of "startReadSms" function.

https://medium.com/@maniac-tech/read-otp-sms-expo-react-native-android-14b702c43623

**Comment by maniac-tech** [comment]
@antoponselvan thanks for sharing the feedback.

Quick question: The Sample repo mentioned in the readme didn't serve this purpose?

**Comment by antoponselvan** [comment]
Hi Abhishek,
Indeed the sample repo has the information that I was originally searching. Seems like, It was a case of me not looking at the sample repo properly. Having said that, I feel the medium article zoomed into the exact information I was looking for quickly. The sample repo has a lot more details.

**Reply by maniac-tech** [reply]
No issues @antoponselvan 

I believe everyone has been in theses situations :) 
If you think we can improve on the visibility please feel free and open a Pull request, I will review and see if we can benefit from them.

**Comment by maniac-tech** [comment]
Closing this discussion. Feel free to open this again, or start a new one if needed.
